benchmarks/src/test/java/io/improbable/keanu/benchmarks/Nd4jDoubleTensorScalarOperationsTest.java

```diff
@@ -0,0 +1,18 @@
+package io.improbable.keanu.benchmarks;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class Nd4jDoubleTensorScalarOperationsTest {
+
+    @Test
+    public void theScalarsHaveTheExpectedRank() {
```

Comment:
Unit tests for the performance tests?

Reply:
The purpose is to make sure that the baseline does what we expect (use a rank 0). If this test fails (which could happen with an ND4J upgrade), it's an indication that the benchmark needs to be rewritten.